src/events/_event-post-macros.html

```diff
@@ -268,31 +268,48 @@ <h2 class="event-venue_heading">{{ city }}</h2>
                     ) }}
                 </div>
                 <div class="content-l_col content-l_col-1-2 event-calendar_container">
-                    {% if (event_state == 'past') %}
+                    {% if (event_state == 'past' and post.archive.youtube is not defined) %}
                         Video Replay will be<br> available shortly
                     {% else %}
                         {{ event_meta_datetime(post.beginning_time) }}
-                        <!-- TDOD: Replace with real download link -->
+                        {% if event_state == 'future' %}
+                        <!-- TODO: Replace with real download link -->
                         <a class="event-calendar_download jump-link jump-link__download" href="#">
                             <span class="jump_link_text">Download .ics</span>
                         </a>
+                        {% endif %}
                     {% endif %}
                 </div>
           </div>
         </header>
         <footer>
             <figure class="event-media_container">
-                {# TODO Replace image URL with actual post url based on event states #}
-                {% if (event_state == 'past')  %}
-                    <img src="https://placeholdit.imgix.net/~text?C3%97350&w=700&h=350" />
-                {% else %}
-                    {{ event_location_image({
-                        'location': city,
-                        'zoom': '12',
-                        'scale': '2',
-                        'size': '640x320'
-                    }) }}
-                {% endif %}
+            <!-- TODO: Add code for live streaming Event and determine image property name -->
+            {% if (event_state == 'past' and post.archive.youtube is defined)  %}
+                {% import 'macros/video-player.html' as video_player %}
+                {% set video_id   = post.archive.youtube.split('=') | last %}
+                {% set image_url  = post.attachments[1].url.replace('content.','') %}
```

Comment:
Checking with @sarahsimpson09 to see how she uploaded image attachments. Don't merge before confirmation.